daemon/assets/ghc-specter.css

```diff
@@ -180,3 +180,59 @@ ul.tree span::before {
     width: 10px;
     background: black;
 }
+
+
+.core-expr {
```

Comment:
In the end, all these should be encapsulated into web components using shadow dom.